ledger/participant-integration-api/src/main/scala/platform/store/dao/HikariJdbcConnectionProvider.scala

```diff
@@ -96,7 +96,9 @@ object HikariConnection {
     )
 }
 
-class HikariJdbcConnectionProvider(dataSource: HikariDataSource, healthPoller: Timer)(
+private[platform] class HikariJdbcConnectionProvider(
+    dataSource: HikariDataSource,
+    healthPoller: Timer)(
     implicit logCtx: LoggingContext
 ) extends JdbcConnectionProvider {
```

Comment:
Scalafmt probably won't like this exact formatting, but basically, can you add a comma?

```suggestion
private[platform] class HikariJdbcConnectionProvider(
    dataSource: HikariDataSource,
    healthPoller: Timer,
  )(implicit logCtx: LoggingContext)
  extends JdbcConnectionProvider {
```